### PR TITLE
Fixes #15025 - do not show foreman_proxy startup messages.

### PR DESCRIPTION
**Pull request by dmitri-d**

### CODE REVIEW

File: lib/proxy/plugins.rb [hunk 1]
```diff
@@ -12,11 +12,22 @@ def self.instance
   end
 
   def plugin_loaded(a_name, a_version, a_class)
-    self.loaded += [{:name => a_name, :version => a_version, :class => a_class, :state => :uninitialized}]
+    self.loaded += [{:name => a_name, :version => a_version, :class => a_class, :state => :unitialized}]
```
Comment (domcleal) [inline lib/proxy/plugins.rb]: typo?  `uninitialized`

File: lib/proxy/plugin_initializer.rb [hunk 2]
```diff
@@ -7,17 +7,27 @@ class ::Proxy::PluginGroup
 
   def initialize(a_plugin, providers = [], di_container = ::Proxy::DependencyInjection::Container.new)
     @plugin = a_plugin
-    @state = :starting
+    @state = :uninitialized # :uninitialized -> :staring -> :running, or :uninitialized -> :disabled, or :uninitialized -> :starting -> :failed
```
Comment (domcleal) [inline lib/proxy/plugin_initializer.rb]: typo, `staring` => `starting`

**Comment by domcleal** [comment]
The PR also needs a rebase after merging dd2fc60.

**Comment by dmitri-d** [comment]
rebased, fixed misspellings.

File: lib/proxy/plugin_initializer.rb [hunk 3]
```diff
@@ -227,22 +274,12 @@ def merge_settings(provider_settings, main_plugin_settings)
     provider_settings.merge(main_plugin_settings)
   end
 
-  def used_settings(settings)
+  def log_used_settings(settings)
     default_settings = plugin.plugin_default_settings
     sorted_keys = settings.keys.map(&:to_s).sort.map(&:to_sym) # ruby 1.8.7 doesn't support sorting of symbols
     sorted_keys.map {|k| "'%s': %s%s" % [k, settings[k], (default_settings.include?(k) && default_settings[k] == settings[k]) ? " (default)" : ""] }.join(", ")
```
Comment (domcleal) [inline lib/proxy/plugin_initializer.rb]: This isn't getting logged anywhere, I think you intended to move the logger.debug into this method?

**Comment by dmitri-d** [comment]
rebased, used settings are being logged now.

**Comment by domcleal** [comment]
Merged as 18a62b7, thanks @witlessbird.

**Pull request closed without merging**